jobs/integration/validation.py

```diff
@@ -2085,104 +2085,128 @@ async def test_multus(model, tools, addons_model):
         pytest.skip("multus is not deployed")
 
     unit = model.applications["kubernetes-control-plane"].units[0]
+    containerd_app = model.applications["containerd"]
     multus_app = addons_model.applications["multus"]
 
     async def cleanup():
         await run_until_success(
             unit,
-            "/snap/bin/kubectl --kubeconfig /root/.kube/config delete pod multus-test --ignore-not-found",
+            "/snap/bin/kubectl --kubeconfig /root/.kube/config delete pod multus-test --ignore-not-found;"
+            "/snap/bin/kubectl --kubeconfig /root/.kube/config delete subnet attachnet --ignore-not-found;",
+        )
+        await multus_app.reset_config(["network-attachment-definitions"])
+        await containerd_app.reset_config(["no_proxy"])
+
+    async def apply_def(content):
+        remote_path = "/tmp/content.yaml"
+        with NamedTemporaryFile("w") as f:
+            yaml.dump(content, f)
+            await scp_to(
+                f.name, unit, remote_path, tools.controller_name, tools.connection
+            )
+        await run_until_success(
+            unit,
+            f"/snap/bin/kubectl --kubeconfig /root/.kube/config apply -f {remote_path}; rm {remote_path}",
         )
-        await multus_app.set_config({"network-attachment-definitions": ""})
 
     await cleanup()
 
-    # Create NetworkAttachmentDefinition for Flannel
+    await containerd_app.set_config(
+        {
+            "no_proxy": "localhost,127.0.0.1,::1,10.246.154.0/24,10.152.183.0/24,192.168.0.0/16"
```

Comment:
This could cause problems if we ever want to run this test manually in environments other than our vSphere lab.

Could this proxy config be set in the deployment overlay in ovn-multus-spec instead? I think it would be nice if we can consolidate our environment-specific configuration there so that test code like this can be less destructive.